### ISSUE TITLE
[CM9 Final 3 / Galaxy Gio] DSPManager

### ISSUE THREAD

**Issue by hsfdmr** [MaclawStudio/aosp#221]
Settings in the DSPManager don´t have any hearable consequences.

For example (all of the following settings together OR a part of them seperatly won´t work for me):
- Enable --> Bass Boost is enabled
- Select Effect Strenght --> Extreme
- Enable --> Equalizer is enabled
- Select Preset --> Bass Booster

**Comment by svigi** [comment]
It worked fine in CM9 Final 2 or 2.1 with Galaxy Gio.
So all you have to do is to replace the /system/etc/audio_effects.conf file from the old rom.
Before that don't forget to remount your /system partition writable.
I don't really remember if I replaced some of the audio librarys, too.
Cheers :)

**Comment by gbr161** [comment]
Yep i cant hear any changes using dsp manager using CM9 final3 at Galaxy Ace :/
